### ISSUE TITLE
Query: lobed nucleus  

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#20764]
Hello, 

This arises from #20643

@ValWood asked to put a taxon constraint on GO:0098537 lobed nucleus, which is defined as "Nucleus with two or more lobes connected by a thin filament that contains no internal chromatin. Examples include the nuclei of mature basophils, eosinophils and neutrophils in mice and humans. ", so I was thinking to add a subclass or a logical definition 'part of' some 'leukocyte' 

Current LD is
intersection_of: GO:0005634 ! nucleus
intersection_of: bearer_of PATO:0001979 ! lobed

@cmungall Is this correct ? 

All uses of 'bearer_of' are a bit odd and should be reviewed. 

Thanks, Pascale

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete GO:0098537 lobed nucleus. The reason for the obsoletion is that this represents the nucleus of specific cell types (basophils, eosinophils and neutrophils). The lobed nuclear appear to be due to differently packaged chromatin, see https://sciencing.com/lobes-nucleus-20628.html, they do not represent different types of nucleus. 

There are no annotations to this term. This term is not present in any subsets. 

You can comment on the ticket here: https://github.com/geneontology/go-ontology/issues/20764

Thanks, Pascale

**Comment by pgaudet** [comment]
Note that this term is used by CL:0000043 and CL:0000096
@addiehl

**Comment by dosumis** [comment]
> Note that this term is used by CL:0000043 and CL:0000096.

Didn't notice an obsoletion notice.  Was there an email?  

This obsoletion is causing problems for the CL release.   Does it do any harm to keep some CC classes defined by morphology?  They can be very useful for defining cell types.

**Comment by pgaudet** [comment]
There was an email to go-friends on 9 March 2021 
and it was announced in the announcements repo - https://github.com/geneontology/go-announcements/issues/203

I'll put this on the editors discussion, but meanwhile, can you use PATO in CL to describe this, as it was described in GO, ie 

intersection_of: GO:0005634 ! nucleus
intersection_of: bearer_of PATO:0001979 ! lobed

? 

Thanks, Pascale

**Comment by cmungall** [comment]
@pgaudet's proposed fix was added as a PR 6 days ago https://github.com/obophenotype/cell-ontology/pull/1131/, I approved this, so I think there is no longer an immediate issue

Note I am skeptical of the correctness of the CL logical definition and of the utility attempting to capture this morphology in OWL, but that's another matter.

Given the tight intercoupling of GO, CL, and Uberon we should be more cautious with obsoletions in future. GO followed the current SOP by using the announcements repo. However, CL has far fewer curator resources than GO and no one is currently assigned the task of monitoring the repo, opening Protege every time there is a notice, and manually checking the usage tab for every single term.

I opened a new ticket to make the script that GO and others can use to improve their obsoletion SOP: 
https://github.com/geneontology/go-ontology/issues/21723
